scripts/etc/init.d/rcS

```diff
@@ -85,6 +85,25 @@ sanitize_inputfile()
     fi
 }
 
+install_files()
+{
+  file_to_read=$1
+  sanitize_inputfile file_to_read
+  grep -v "^[[:space:]]*#\|^[[:space:]]*$" "${file_to_read}" | while read -r line; do
+    owner=$(echo "$line" | awk '{ print $1 }')
+    perms=$(echo "$line" | awk '{ print $2 }')
+    file=$(echo  "$line" | awk '{ print $3 }')
+    if [ ! -d "config/files$file" ]; then
+      mkdir -p "/rootfs$(dirname "$file")"
+      cp "config/files$file" "/rootfs$file"
+    else
+      mkdir -p "../rootfs/$file"
```

Comment:
it seems you really like the double dots :wink: 
but I don't think `..` should be here